reinforcement_learning/rl_cartpole_ray/src/train-rl-cartpole-ray.py

```diff
@@ -26,6 +26,7 @@ def get_experiment_config(self):
                 "training_iteration": 40
             },
             "config": {
+              "use_pytorch": False,
```

Comment:
If "use_pytorch" by default is `False`, it should be by default in the launcher code, not in the config

Reply:
This is part of the rllib experiment config. Launcher reads it via ` use_pytorch = experiment_config["training"]["config"].get("use_pytorch", False)`

Reply:
If we use `get("use_pytorch", False)` we don't need to explicitly set it as `False` in the config

Reply:
Correct. It's more like an illustration of how a customer can switch between frameworks easily with RLlib on SageMaker.